openehr-rm/src/main/java/com/nedap/archie/rminfo/ArchieRMInfoLookup.java

```diff
@@ -6,33 +6,70 @@
+import com.nedap.archie.rm.archetyped.TemplateId;
+import com.nedap.archie.rm.changecontrol.Contribution;
+import com.nedap.archie.rm.changecontrol.ImportedVersion;
+import com.nedap.archie.rm.changecontrol.OriginalVersion;
+import com.nedap.archie.rm.changecontrol.Version;
+import com.nedap.archie.rm.changecontrol.VersionedObject;
 import com.nedap.archie.rm.composition.*;
-import com.nedap.archie.rm.changecontrol.*;
-
 import com.nedap.archie.rm.datastructures.*;
-import com.nedap.archie.rm.datatypes.*;
-import com.nedap.archie.rm.datavalues.encapsulated.*;
-import com.nedap.archie.rm.datavalues.quantity.datetime.*;
-import com.nedap.archie.rm.datavalues.quantity.*;
+import com.nedap.archie.rm.datatypes.CodePhrase;
 import com.nedap.archie.rm.datavalues.*;
-import com.nedap.archie.rm.*;
-import com.nedap.archie.rm.datavalues.timespecification.*;
+import com.nedap.archie.rm.datavalues.encapsulated.DvEncapsulated;
+import com.nedap.archie.rm.datavalues.encapsulated.DvMultimedia;
+import com.nedap.archie.rm.datavalues.encapsulated.DvParsable;
+import com.nedap.archie.rm.datavalues.quantity.*;
+import com.nedap.archie.rm.datavalues.quantity.datetime.DvDate;
+import com.nedap.archie.rm.datavalues.quantity.datetime.DvDateTime;
+import com.nedap.archie.rm.datavalues.quantity.datetime.DvDuration;
+import com.nedap.archie.rm.datavalues.quantity.datetime.DvTemporal;
+import com.nedap.archie.rm.datavalues.quantity.datetime.DvTime;
+import com.nedap.archie.rm.datavalues.timespecification.DvGeneralTimeSpecification;
+import com.nedap.archie.rm.datavalues.timespecification.DvPeriodicTimeSpecification;
+import com.nedap.archie.rm.datavalues.timespecification.DvTimeSpecification;
 import com.nedap.archie.rm.demographic.*;
-import com.nedap.archie.rm.directory.*;
-import com.nedap.archie.rm.generic.*;
-import com.nedap.archie.rm.ehr.*;
-import com.nedap.archie.rm.integration.*;
-import com.nedap.archie.rm.security.*;
+import com.nedap.archie.rm.directory.Folder;
+import com.nedap.archie.rm.directory.VersionedFolder;
+import com.nedap.archie.rm.ehr.Ehr;
+import com.nedap.archie.rm.ehr.EhrAccess;
+import com.nedap.archie.rm.ehr.EhrStatus;
+import com.nedap.archie.rm.ehr.VersionedComposition;
+import com.nedap.archie.rm.ehr.VersionedEhrAccess;
+import com.nedap.archie.rm.ehr.VersionedEhrStatus;
+import com.nedap.archie.rm.generic.Attestation;
+import com.nedap.archie.rm.generic.AuditDetails;
+import com.nedap.archie.rm.generic.Participation;
+import com.nedap.archie.rm.generic.PartyIdentified;
+import com.nedap.archie.rm.generic.PartyProxy;
+import com.nedap.archie.rm.generic.PartyRelated;
+import com.nedap.archie.rm.generic.PartySelf;
+import com.nedap.archie.rm.generic.RevisionHistory;
+import com.nedap.archie.rm.generic.RevisionHistoryItem;
+import com.nedap.archie.rm.integration.GenericEntry;
+import com.nedap.archie.rm.security.AccessControlSettings;
 import com.nedap.archie.rm.support.identification.*;
-import com.nedap.archie.rm.support.*;
```

Comment:
This package does not exist by itself, only as part of `com.nedap.archie.rm.support.identification`. Java 11 stopped accepting this, so simply removed it. The rest is the result of a optimize imports operation, and not strictly required.